modules/check_in/app/controllers/check_in/v2/appointments_controller.rb

```diff
@@ -44,6 +45,15 @@ def merge_facilities(appointments)
         end
       end
 
+      def merge_clinic(appointments)
+        appointments.each do |appt|
+          unless appt[:locationId].nil? && appt[:clinic].nil?
+            appt[:clinic_info] =
```

Comment:
if all other attributes are in camelCase, maybe this should be too.

Reply:
Please find the updates in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)